pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```diff
@@ -203,4 +203,12 @@ in {
   fourmolu = overrideCabal (drv: {
     libraryHaskellDepends = drv.libraryHaskellDepends ++ [ self.file-embed ];
   }) (disableCabalFlag "fixity-th" super.fourmolu);
+  # haskell-gi
```

Comment:
You'll want to use `__CabalEagerPkgConfigWorkaround` for all of these like this:

https://github.com/NixOS/nixpkgs/blob/d05e6727a2b511106ace4e08a9ce2a5cf17dde92/pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix#L203-L210